press.html

```diff
@@ -72,17 +72,18 @@ <h3>In other languages</h3>
 
       <p><h4><a href="./files/jyllendsposten.pdf">Flere kvinder til computeren,tak</a></h4>Jyllands Posten 13.4.2012</p>
 
-            <blockquote class="quotes">
-      It’s events like these that have the power to change the face of the tech world, enabling not just women but anyone to try their hand at coding and building something great.
-            </blockquote>
+      <blockquote class="quotes">
+        It's events like these that have the power to change the face of the tech 
+        world, enabling not just women but anyone to try their hand at coding and building something great.
+      </blockquote>
 
       <blockquote class="quotes">
-        I'm so excited to learn Ruby on Rails and I my heart leapt when I heard
+        I'm so excited to learn Ruby on Rails - my heart leapt when I heard
         that you guys were organising this workshop!
       </blockquote>
 
       <blockquote class="quotes">
-        I work as journalist. I think I and most of us journalists (especially women)
+        I work as journalist. I think most of us journalists (especially women)
```

Comment:
Actually, just re-reading this, I don't think this is a typo after all... but, grammatically, it doesn't make sense without commas. You can either remove `I and` (and it'll make sense again), or add commas in so that it reads `I think I, and most of us journalists (especially women), understand far too little about computers` - it's up to you.